package.json

```diff
@@ -24,9 +24,9 @@
     "url": "https://github.com/bigcommerce/bigcommerce-api-node.git"
   },
   "dependencies": {
-    "axios": "^0.26.1",
+    "axios": "^0.27.2",
     "form-data": "^4.0.0",
-    "jsonwebtoken": "^8.5.1"
+    "jsonwebtoken": "^9.0.0"
```

Comment:
There were a number of breaking changes between v8 and v9: 
> Breaking changes: See [Migration from v8 to v9](https://github.com/auth0/node-jsonwebtoken/wiki/Migration-Notes:-v8-to-v9)

Did you have a chance to confirm that our implementation is not affected?

Reply:
This works.  I made this change with the prior closed PR I had for this ticket.